modules/InboundEmail/InboundEmail.php

```diff
@@ -4164,7 +4164,7 @@ public function handleTranserEncoding($str, $enc = 0)
      */
     public function getMessageId($header)
     {
-        $message_id = md5(print_r($header, true));
+        $message_id = md5( );
```

Comment:
I don't really understand why we removed the unique header from this function, it was an intended direction or just accidentally?